src/ManageCourses.Api/Controllers/PublishController.cs

```diff
@@ -31,13 +31,16 @@ public PublishController(IDataService dataService, IEnrichmentService enrichment
         [ProducesResponseType(typeof(bool), 200)]
         [ProducesResponseType(400)]
         [ProducesResponseType(404)]
-        public async Task<ActionResult> PublishToSearchAndCompare(string instCode, string courseCode)
```

Comment:
this should be configured from an environment variable rather than a parameter

Reply:
Why would we want the setting in two places?  The Ui has the new feature flag. This setting is passed then into the Api via the parameter.  This endpoint doing is doing 2 things here: 1 changing the Status and 2. Sending to SearchAndCompare. Previously we had no control over this. Also the return value from the SearchAndCompare service is totally ignored so it's really a 'fire and forget' action. The parameter seems to me the best option! Happy to be overruled though.